src/scripts/main.js

```diff
@@ -1,3 +1,247 @@
 'use strict';
 
-// write code here
+/* eslint-disable */
+const body = document.querySelector('body');
+const tHeadCells = document.querySelectorAll('table thead th');
+const tbody = document.querySelector('table tbody');
+const rows = [...tbody.querySelectorAll('tr')];
+
+const officeOptions
+  = ['Tokyo', 'Singapore', 'London', 'New York', 'Edinburgh', 'San Francisco'];
+
+employeeForm();
+
+for (let i = 0; i < tHeadCells.length; i++) {
```

Comment:
Replace for loop with forEach, e.g. tHeadCells.forEach((cell, index) => { ... })